openedge-checks/src/main/java/org/sonar/plugins/openedge/api/checks/OpenEdgeProparseCheck.java

```diff
@@ -115,7 +112,7 @@ public void reportIssue(InputFile file, JPNode node, String msg) {
    */
   public void reportIssue(InputFile file, String fileName, int lineNumber, String msg) {
     LOG.trace("Adding issue {} to {} line {}",
-        new Object[] {getRuleKey() == null ? null : getRuleKey().rule(), fileName, lineNumber});
+        getRuleKey() == null ? null : getRuleKey().rule(), fileName, lineNumber);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Invoke method(s) only conditionally. Use the built-in formatting to construct this argument. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS2629)